### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amaljose02**
Added  Table of contents

## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [x] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number:  NA

## What is the current behavior?
 NA
## What is the new behavior?
NA
## Other information
NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +52,11 @@ _Note: If coming from a deb install, the directory structure is different and yo
 | Latest Release | [FlatHub Page](https://flathub.org/apps/details/io.lbry.lbry-app)         | [AUR Package](https://aur.archlinux.org/packages/lbry-app-bin/)                           | [Pi Installer](https://lbrypi.com)          |
 | Maintainers    | [@choofee](https://github.com/choffee)/[@iuyte](https://github.com/iuyte) | [@kcseb](https://github.com/kcseb)/[@TimurKiyivinski](https://github.com/TimurKiyivinski) | [@Madiator2011](https://github.com/kodxana) |
 
-## Usage
+<h2 align="left">Usage</h2>
```
Comment (seanyesmunt) [inline README.md]: We shouldn't need to change these, the links will still work.

Comment (amaljose02) [reply README.md]: I have changed the `h2` to `##`.

**Comment by tzarebczan** [comment]
@amaljose02, thanks for the PR! 

Can we show you some [appreciation](https://lbry.com/faq/appreciation) for the contribution? 

Also, we're giving away some [Hacktoberfest](https://lbry.com/news/hacktoberfest-2019) bonuses and goodies for this month. We'll get it all figured out after you shoot us an email after this is reviewed/merged.

**Comment by amaljose02** [comment]
@tzarebczan thanks for the appreciation. Could you please tell to which mail should I send the details for receiving the goodies.

**Comment by amaljose02** [comment]
Hi @tzarebczan , It will be very helpful if you share  the details to contact to receive the goodies and bonuses?

**Comment by tzarebczan** [comment]
@amaljose02 sorry for not replying earlier, but the links were part of the original reply. Since the promotion is now over, just get us an email over and we'll get some credits over to you for the PR.

**Pull request merged**